src/Common/Services/IEnvironmentDeploymentHandler.cs

```diff
@@ -7,5 +7,7 @@ namespace Cmf.CustomerPortal.Sdk.Common.Services
     public interface IEnvironmentDeploymentHandler
     {
         Task Handle(bool interactive, CustomerEnvironment customerEnvironment, string deploymentTarget, DirectoryInfo outputDir);
+
+        Task<bool> WaitForEnvironmentsToBeTerminated(CustomerEnvironmentCollection customerEnvironments);

```

Comment:
Boolean return value does not seem to be meaningful (in case of timeout an exception is thrown; otherwise, its value is always true)

Reply:
true, it makes no sense